bundles/org.openhab.binding.icloud/src/test/java/org/openhab/binding/icloud/TestICloud.java

```diff
@@ -73,7 +74,10 @@ public void testAuth() throws IOException, InterruptedException, ICloudApiRespon
         ICloudService service = new ICloudService(iCloudTestEmail, iCloudTestPassword, stateStorage);
         service.authenticate(false);
         if (service.requires2fa()) {
-            System.out.print("Code: ");
+            PrintStream consoleOutput = System.out;
```

Comment:
Is it really technically possible that stdout is NULL?
What about System.in (next lines)?

Reply:
Don’t think it will ever be null. But it is nullable and the compiler warned about it so I added this check.

Reply:
OK -1 for readabillity ;-), that the problem with those static code analyzer, they are often a bit silly. But I can live with it.